gnucash/gnome/dialog-vendor.c

```diff
@@ -211,23 +211,10 @@ gnc_vendor_window_ok_cb (GtkWidget *widget, gpointer data)
 
     /* Check for valid company name */
     if (check_entry_nonempty (vw->company_entry,
-                              _("You must enter a company name. "
-                                "If this vendor is an individual (and not a company) "
-                                "you should enter the same value for:\nIdentification "
-                                "- Company Name, and\nPayment Address - Name.")))
+                              _("The Company Name field cannot be left blank,\n"
```

Comment:
What is the purpose of the hard linebreak here?

Reply:
Just keep the length of line same as before.
Now I find it's unnecessary.

how about this:
"The Company Name field cannot be left blank, please "
 "enter a company name or a person's name."

Should I close this PR , and creat a new one?

Reply:
No need for a new PR, just amend the commit and force push:
```
git commit -a --amend
git push -f my-repo maint
```
(substituting your actual remote for 'my-repo' of course).

Reply:
Thanks for your patient guidance. 
I have added another commit.